exonum/src/helpers/fabric/maintenance.rs

```diff
@@ -114,9 +114,10 @@ impl Command for Maintenance {
             .arg::<String>(MAINTENANCE_ACTION_PATH)
             .unwrap_or_else(|_| panic!("{} not found.", MAINTENANCE_ACTION_PATH));
 
-        match action.as_ref() {
-            "clear-cache" => Self::clear_cache(&context),
-            _ => println!("Unsupported maintenance action: {}", action),
+        if action == "clear-cache" {
```

Comment:
This change has been suggested by Clippy, right?
(I don't like it by the way. :upside_down_face: )

Reply:
Exactly. Clippy is guilty